src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md

```diff
@@ -1,15 +1,56 @@
 ---
-title: How to Create or Delete a User or Sudo User on Ubuntu
+title: How to Create/Delete a User or Sudo User on Linux
```

Comment:
What about a simple `User management on Linux` ?

Reply:
It's also in there.

Reply:
That's good.

Reply:
Fixed the title along with folder name